tests/fixtures/with_default_source_and_pypi/pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[tool.poetry]
+name = "with-default-source-and-pypi"
+version = "1.2.3"
+description = "Some description."
+authors = [
+    "Sébastien Eustace <sebastien@eustace.io>"
+]
+license = "MIT"
+
+readme = "README.rst"
+
+homepage = "https://python-poetry.org"
+repository = "https://github.com/python-poetry/poetry"
+documentation = "https://python-poetry.org/docs"
+
+keywords = ["packaging", "dependency", "poetry"]
+
+classifiers = [
+    "Topic :: Software Development :: Build Tools",
+    "Topic :: Software Development :: Libraries :: Python Modules"
+]
+
+# Requirements
+[tool.poetry.dependencies]
+python = "~2.7 || ^3.6"
```

Comment:
I am aware that this was most likely copied from another fixture, but could you remove the reference to Python 2? I want to completely remove it one day, so let's not add more of it :P